src/module/actor/data-model-character.js

```diff
@@ -72,15 +77,16 @@ export default class OseDataModelCharacter extends foundry.abstract.DataModel {
       hp: new ObjectField({
         hd: new StringField(),
         value: new NumberField({ integer: true }),
-        max: new NumberField({ integer: true })
+        max: new NumberField({ integer: true }),
       }),
       thac0: new ObjectField(),
-      initiative: new ObjectField(),
```

Comment:
why is this missing?

Reply:
The linter pointed it out as a duplicate.